spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -108,7 +108,8 @@ def __init__(self, parent, id_,
                  ask_before_closing=False,
                  css_path=None,
                  configuration=None,
-                 handlers={}):
+                 handlers={},
+                 std_dir=None):

```

Comment:
```suggestion
                 handlers={}):
```

I think this change is not necessary for this PR.